res/openvpn-gui-res-zh-hans.rc

```diff
@@ -254,19 +254,19 @@ END
 /* URL Profile Import Dialog */
 ID_DLG_URL_PROFILE_IMPORT DIALOG 6, 18, 249, 95
 STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CENTER | DS_SETFOREGROUND
-CAPTION "Import Profile from Access Server"
+CAPTION "从数据服务器导入配置文件"
```

Comment:
"Access Server" is the product name of OpenVPN Inc's commercial offering --- not to be translated or add (Access Server) in parenthesis? Here and one instance below.

Reply:
Sorry, I shouldn't translate here.
Thank you for reminding me